subsys/bluetooth/mesh/access.c

```diff
@@ -404,15 +404,19 @@ struct bt_mesh_elem *bt_mesh_elem_find(u16_t addr)
 {
 	int i;
```

Comment:
just rename var `int i` change `u16_t  index`.
remove late define `u16_t index`.

Reply:
@trond-snekvik do you want to go with the suggestion from @menglingao1997 to have one less stack variable?

Reply:
Done